extensions/Gadgets/Gadgets_body.php

```diff
@@ -222,7 +223,20 @@ private static function applyScript( $page, $out ) {
 			return;
 		}
 
-		$u = $t->getLocalURL( 'action=raw&ctype=' . $wgJsMimeType );
+		// Wikia change begin; author: lukaszk
+		$extraQuery = '';
+
+		if ( $wgEnableContentReviewExt ) {
+			$contentReviewHelper = new Wikia\ContentReview\Helper();
+
+			if ( !$contentReviewHelper->isContentReviewTestModeEnabled() ) {
+				$hash = $contentReviewHelper->getSiteJsScriptsHash();
+				$extraQuery = '&reviewed=' . $hash;
+			}
+		}
+
+		$u = $t->getLocalURL( 'action=raw&ctype=' . $wgJsMimeType . $extraQuery );
+		// Wikia change end
```

Comment:
I'll create a ticket for it but for now I wouldn't treat it as a release blocker for beta tests. Are you ok with it? :)